src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1565,45 +1565,12 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+            <include cluster="Binding" client="false" server="true" clientLocked="true" serverLocked="true">
```

Comment:
The conformance for Binding is `[Matter]` which means it _may_ be supported for Matter

Reply:
Updated to optional both Binding and Time sync